src/test/java/htsjdk/tribble/TestUtils.java

```diff
@@ -18,11 +18,30 @@
 
 package htsjdk.tribble;
 
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.FileSystem;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 
 /**
  * User: jacob
  * Date: 2012-Dec-13
  */
 public class TestUtils {
     public static String DATA_DIR = "src/test/resources/htsjdk/tribble/";
+
+
+    public static Path getTribbleFileInJimfs(String vcf, String index, FileSystem fileSystem) throws IOException, URISyntaxException {
+        final FileSystem fs = fileSystem;
+        final Path root = fs.getPath("/");
+        final Path vcfPath = Paths.get(vcf);
+        if (index != null) {
+            final Path idxPath = Paths.get(index);
+            final Path idxDestination = Paths.get(AbstractFeatureReader.isTabix(vcf, index) ? Tribble.tabixIndexFile(vcf) : Tribble.indexFile(vcf));
+            Files.copy(idxPath, root.resolve(idxDestination.getFileName().toString()));
```

Comment:
You can resolve with the root the __vcfPath__ for the `JimfsFileSystem` before, and use the Tribble methods for get the idxDestination name without the need to resolve it again against the root.

Reply:
Thanks. that's a bit simpler.